src/test/java/com/google/ehub/servlets/LoginServletTest.java

```diff
@@ -75,12 +67,12 @@ private JsonObject getLoginServletResponse() throws IOException {
     String responseStr = stringWriter.getBuffer().toString().trim();
     JsonElement responseJsonElement = new JsonParser().parse(responseStr);
     JsonObject responseJsonObject = responseJsonElement.getAsJsonObject();
-    
+
     return responseJsonObject;
   }
 
   @Test
-  public void getRequestWithLoggedInUser_validContentGetsSent() throws IOException {
+  public void getRequestWithLoggedInUser_sendsLogoutUrl() throws IOException {
     JsonObject responseJsonObject = getLoginServletResponse();
 
     LOGIN_URL = responseJsonObject.get("LoginURL").getAsString();
```

Comment:
Below is an example of how to replace the class fields `LOGIN_URL` and `LOGOUT_URL` with local variables. You can do something similar in the test case below too.
These lines can be replaced with:
```
    String loginUrl = responseJsonObject.get("LoginURL").getAsString();
    String logoutUrl = responseJsonObject.get("LogoutURL").getAsString();	
    Assert.assertTrue(logoutUrl.contains("logout"));
    Assert.assertTrue(loginUrl.isEmpty());
```

Reply:
I fixed it to have this change now, will push